mmv1/third_party/validator/tests/data/example_folder_iam_member_empty_folder.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "name": "//cloudresourcemanager.googleapis.com/placeholder-c2WD8F2q",
```

Comment:
should this be `organizations/unknown`?

Reply:
this should be the folder name, in the forms of folders/xxx, but that value is empty, so end up with a placeholder